src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTests.vb

```diff
@@ -11741,16 +11739,15 @@ End Class
 </compilation>
 
             Dim compilation = CreateCompilationWithMscorlibAndVBRuntime(source, TestOptions.ReleaseExe)
-
-            AssertTheseDiagnostics(compilation,
-<expected>
-BC40054: 'Public Sub New(a As Integer)' in designer-generated type 'FromDesigner1' should call InitializeComponent method.
-    Sub New(a As Integer)
-        ~~~
-BC40054: 'Public Sub New(c As Integer)' in designer-generated type 'FromDesigner5' should call InitializeComponent method.
-    Sub New(c As Integer)
-        ~~~
-</expected>)
```

Comment:
Sure. I will use AssertTheseEmitDiagnostics to keep those original forms. Thanks!